src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -912,4 +912,46 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="ConfigGetCommandDescription" xml:space="preserve">
     <value>Gets the NuGet configuration settings that will be applied.</value>
   </data>
+  <data name="WhyCommand_Description" xml:space="preserve">
+    <value>Lists the dependency graph for a particular package for a given project or solution.</value>
+  </data>
+  <data name="WhyCommand_PathArgument_Description" xml:space="preserve">
+    <value>A path to a project, solution file, or directory.</value>
+  </data>
+  <data name="WhyCommand_PackageArgument_Description" xml:space="preserve">
+    <value>The package name to lookup in the dependency graph.</value>
+  </data>
+  <data name="WhyCommand_FrameworksOption_Description" xml:space="preserve">
+    <value>The target framework(s) for which dependency graphs are shown.</value>
+  </data>
+  <data name="WhyCommand_Error_ArgumentCannotBeEmpty" xml:space="preserve">
+    <value>Unable to run 'dotnet nuget why'. The '{0}' argument cannot be empty.</value>
+    <comment>{0} - Argument that was not provided</comment>
+  </data>
+  <data name="WhyCommand_Error_PathIsMissingOrInvalid" xml:space="preserve">
+    <value>Unable to run 'dotnet nuget why'. Missing or invalid project/solution file '{0}'.</value>
+    <comment>{0} - Project/solution file path</comment>
+  </data>
+  <data name="WhyCommand_Error_InvalidFramework" xml:space="preserve">
+    <value>Failed to parse one of the given frameworks. Please make sure the given frameworks are valid. For a list of accepted values, please visit https://learn.microsoft.com/en-us/nuget/reference/target-frameworks#supported-frameworks.</value>
```

Comment:
I've removed the framework validation since we match on target aliases instead. Now, we're not doing any validation on the input frameworks strings. Let me know if I should add any validations for these inputs